src/models/goalTemplate.js

```diff
@@ -59,6 +59,10 @@ export default (sequelize, DataTypes) => {
       allowNull: true,
       type: DataTypes.DATE,
     },
+    deletedAt: {
```

Comment:
This is not needed when paranoid is set to true.

Reply:
oh ok nice